handler.go

```diff
@@ -536,27 +567,36 @@ func (fm *Frontman) runServiceCheck(check ServiceCheck) (map[string]interface{},
 	case <-done:
 		return results, err
 	case <-time.After(serviceCheckEmergencyTimeout):
-		log.Errorf("serviceCheck: %s got unexpected timeout after %.0fs", check.UUID, serviceCheckEmergencyTimeout.Seconds())
-		return nil, fmt.Errorf("got unexpected timeout")
+		secs := serviceCheckEmergencyTimeout.Seconds()
+		checkLog.Errorf("serviceCheck: got unexpected timeout after %.0fs", secs)
+		return nil, errors.New("got unexpected timeout")
 	}
 }
 
 func (fm *Frontman) processInput(input *Input, resultsChan chan<- Result) {
-	wg := sync.WaitGroup{}
```

Comment:
As with string comparisons using `len()`, please don't make unnecessary code changes just to fit your style

Reply:
This change is not because of style. WaitGroup as well as Mutex are locks, and by initialising a variable by value you're risking to end up with accidental copy of it, especially when passing down in arguments of goroutines or structs. Such case can never happen with `*sync.WaitGroup` instead. I don't care about all cases, but this particular waitgroup is used in piece of code I changed below, so I don't want my changes to fail in some implicit way later.

Reply:
gotcha, makes sense 👍